packages/react/src/patterns/blocks/CalloutWithMedia/README.md

```diff
@@ -0,0 +1,69 @@
+# Callout With Media
+
+> The "Callout With Media" is a decorator of `Callout`, which includes
+> `Content Block Simple`.
+
+## Getting started
+
+Here's a quick example to get your started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/callout-with-media/index';
+```
+
+> 💡 Only import fonts once per usage. Don't forget to import the
+> CalloutWithMedia styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CalloutWithMedia } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const copy = 'lorum ipsum dolor sit ameet';
+  const heading = 'lorum ipsum dolor sit amet';
+
+  return <CalloutWithMedia heading={heading} copy={copy} />;
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line in your `.env` file at the root of your project.
+[See more details](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/styles#usage).
+
+```
+  SASS_PATH=node_modules:src
+```
+
+## Props
+
+| Name                 | Required | Data Type | Default Value | Description                                                                                                                                                               |
+| -------------------- | -------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `contentblocksimple` | YES      | Object    | null          | Content Block Simple. See [Quote](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/src/patterns/blocks/ContentBlockSimple/README.md) |
```

Comment:
You're referencing `Quote` here. Also, I wouldn't call the prop `contentblocksimple`, but maybe `props for ContentBlockSimple`

Reply:
updating